app/controllers/backend/backend_controller.rb

```diff
@@ -40,21 +40,28 @@ def request_clubnames
     return Club.pluck(:internal_name) if Rails.env.development?
 
     ugent_login = session[:cas]['user']
+
     def digest(*args)
       Digest::SHA256.hexdigest args.join('-')
```

Comment:
256 is a magic number, please abstract!

Also digest function should be set in config, because it can change